src/main/java/racing/InputView.java

```diff
@@ -9,6 +11,21 @@ public InputView(Scanner scanner) {
         this.scanner = scanner;
     }
 
+    public List<String> inputCarNames() {
+        System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).");
+        String carNames = scanner.nextLine();
+        return split(carNames);
+    }
+
+    private List<String> split(String carNames) {
+        List<String> carNameList = new ArrayList<>();
+        String[] carNameArray = carNames.split(",");
+        for (String carName : carNameArray) {
+            carNameList.add(carName.trim());
+        }
+        return carNameList;
```

Comment:
```suggestion
       return  Arrays.stream(carNames.split(","))
                .map(String::trim)
                .collect(Collectors.toList());
```
- 위와 같은 로직은 Stream API를 활용하는 것이 사이드이펙트도 줄이고, 가독성도 좋을거라 생각해요